### ISSUE TITLE
[Accessibility] Narrator doesn't read `loadingText` if there is no item found from BasePicker.

### ISSUE THREAD

**Issue by yumikohey** [OfficeDev/office-ui-fabric-react#7336]
### Environment Information 
Edge browser.

### Please provide a reproduction of the bug in a codepen:
 https://developer.microsoft.com/en-us/fabric#/components/pickers
![image](https://user-images.githubusercontent.com/3360621/49675277-cd506000-fa29-11e8-8e34-3edc1e068167.png)

 
#### Actual behavior:
When there is no item found, the `props.loadingText` is not being read by narrator.
 
#### Expected behavior:
Narrator should read `props.loadingText` when item is not found.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix?  -- Yes

Requested priority: Normal
 
Products/sites affected: (if applicable)

**Comment by yumikohey** [comment]
@kkjeer exactly! That's what I saw from the code base, too :)

**Comment by joschect** [comment]
@yumikohey and @kkjeer , this is definitely working as intended. If the call isn't asynchronous then the loading text should never get read because it's not loading. For example check out https://developer.microsoft.com/en-us/fabric#/components/peoplepicker . You can click on the 'Delay Suggestion Results' toggle to see how it behaves differently when there is actual loading time.

I do not believe a screen reader should say loading in this case because a sighted user doesn't get that information either.

**Comment by kkjeer** [comment]
Thanks @joschect for confirming. Will close since this is by design.
